proto/osmosis/poolmanager/v1beta1/swap_route.proto

```diff
@@ -38,3 +38,34 @@ message SwapAmountOutSplitRoute {
     (gogoproto.nullable) = false
   ];
 }
+
+// There are two Route types defined here: one involving maps and one involving
```

Comment:
Where do we utilize maps in a state entry?

Its factual that iterating over maps is non-deterministic. Additionally, I originally implemented this with maps in state since I wasn't fully aware of this, and I was app hashing due to it.

You will also notice that we do not have any maps in proto files, with the exception of this, since it is being used strictly to have the maps print words rather than memory address locations.